PyRDF/backend/AWS.py

```diff
@@ -154,9 +261,21 @@ def invoke_root_lambda(client, root_range, script):
 
         print(bench)
 
-        return accumulator
+        return reduction_result
         # reduced_output = pickle.loads(result)
         # return reduced_output
 
     def distribute_files(self, includes_list):
         pass
+
+    @staticmethod
+    def process_execution(s3_client, processing_bucket, num_of_lambdas, logger):
```

Comment:
this name is a bit misleading, maybe something more of "wait_for_all_lambdas" or "await_for_completion"